cmd/newCollateralGenerator.go

```diff
@@ -0,0 +1,240 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/makerdao/vdb-mcd-transformers/generators/new_collateral"
+	"github.com/makerdao/vdb-mcd-transformers/generators/new_collateral/config"
+	"github.com/makerdao/vdb-mcd-transformers/generators/new_collateral/helpers"
+	"github.com/makerdao/vdb-mcd-transformers/generators/new_collateral/initializer"
+	"github.com/makerdao/vdb-mcd-transformers/generators/new_collateral/types"
+	"github.com/manifoldco/promptui"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	collateral             types.Collateral
+	medianContractRequired bool
+	osmContractRequired    bool
+)
+
+var addNewCollateralCmd = &cobra.Command{
+	Use:     "addNewCollateral",
+	Short:   "",
+	Long:    ``,
+	Example: ``,
```

Comment:
Don't you want an example? I get the lack of a long and short.

Reply:
Nice call - just pushed an update.